Dockerfile

```diff
@@ -5,13 +5,31 @@ RUN mkdir /app
 WORKDIR /app
 
 COPY poetry.lock pyproject.toml /app/
-RUN poetry install --no-root --no-dev
+RUN poetry config virtualenvs.create false --local && poetry install --no-root --no-dev
 
 COPY . /app/
 
 FROM base as production
 ENV FLASK_ENV=production
-ENTRYPOINT ["poetry", "run", "gunicorn", "todo_app.app:create_app()", "--bind", "0.0.0.0:5000"]
+CMD gunicorn --bind 0.0.0.0:$PORT 'todo_app.app:create_app()'
```

Comment:
This change has broken the production build steps from the readme since $PORT is not defined. We should be able to either specify a default argument here in the Dockerfile, or pass one through as part of your docker-compose

I think unrelated to these changes, but your local dev container also seems to be slightly broken - I think because the todo_items.py file sits in your root folder and isn't copied into the dev container. Might be worth fixing that while you're here